### DISCUSSION TITLE
Can I get the ID of the edge when I drag the new node to the edge?

### DISCUSSION THREAD

**Discussion by Eliwang4264** [xyflow/xyflow#4600] [Q&A] [answered]
In the Drag and Drop example, can I get the ID of the edge when I drop the node on the edge?
<img width="365" alt="WXWorkCapture_17248074799432" src="https://github.com/user-attachments/assets/fb7603b5-94aa-476f-b878-51c7f269cdbd">

**Comment by bcakmakoglu** [comment, marked as answer]
It is possible, yeah.
[Here's an example.](https://codesandbox.io/p/sandbox/strange-sammet-dc46ft?file=%2FApp.js%3A79%2C1)

Although it's not gonna work properly (in the example I created) if you don't drop properly onto the edge.
But you can use that as a starting point and probably refine it a bit :)

**Reply by Eliwang4264** [reply]
oh , Thank you very much for your reply. Your example fully meets my needs and is very helpful to me
